### ISSUE TITLE
Rack View - Devices

### ISSUE THREAD

**Issue by nicko170** [digitalocean/netbox#317]
A 44U device placed in U1 shows up in U44. Would expect per notes in device creation to show up in U1, however would like for it to show in the whole rack instead of just a disabled field for the rest of the RU in that rack.

(this whole front view would be green)

![image](https://cloud.githubusercontent.com/assets/9477420/16907295/cc3294a0-4d05-11e6-9358-c4a523b38154.png)

**Comment by jeremystretch** [comment]
This is because we currently only define CSS classes for up to 42U in height. We could add more, but maybe it make sense to simply set the height and padding attributes directly on the elements within the template.

Out of curiosity, what sort of device do you have that's 44U in height?

**Comment by Zanthras** [comment]
My most common example of devices taking the entire cab, are storage arrays, single monolithic enterprise sized storage arrays.

**Comment by nicko170** [comment]
Thanks! This is now working.

Full rack colocation - so they can be searched from here. I've created generic colocation for 1RU - 45RU as a device which works well as I can then link to switches and power too :+1:    (unless there is a better way to do this - but this seems to be working well) and also the reason I opened #304 to get an overview of used rackspace when looking to provision new services.
